### ISSUE TITLE
SA1636 - Build 1.1.0-beta001 - Breaks Copyright / Header Validation

### ISSUE THREAD

**Issue by dmccaffery** [DotNetAnalyzers/StyleCopAnalyzers#2330]
## Upgrading to `1.1.0-beta001` from `1.0.0` causes SA1636 errors

```
warning SA1636: The file header copyright text must match the copyright text from the settings.
```

### StyleCop Settings

The stylecop.json file remains unchanged:

```json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "companyName": "automotiveMastermind and contributors",
      "copyrightText": "© automotiveMastermind and contributors. Licensed under MIT. See LICENSE and CREDITS for details.",
      "headerDecoration": "---------------------------------------------------------------------------------------------------------------------"
    }
  }
}
```

I was interested in the `headerDecoration` support. It does work just fine from VS 2017, but is broken with a `dotnet build` at the cli.

### dotnet-cli Info

```bash
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /Users/dmccaffery/.dotnet/sdk/1.0.1
```

**Comment by sharwell** [comment]
@dmccaffery Is this a duplicate of Microsoft/msbuild#1925? In other words, does this new warning appear in *all* files, or just in the generated **AssemblyInfo.cs** files?

**Comment by dmccaffery** [comment]
It was everything if memory serves me correctly. It seems to be resolved, since I closed the associated ticket. I will try to reproduce again later today and will report back.

I disable assembly info generation as condo (the build system I use) does it’s own, so it wasn’t just that.

**Comment by sharwell** [comment]
@dmccaffery I'm trying to get 1.1.0-beta002 out the door since beta001 doesn't include the JSON parsing changes that we implemented for the 1.0.2 patch. Let me know the result of your test, and hopefully if anything isn't working the way you expect then it's just something small that the next release corrects.
